src/run.ts

```diff
@@ -57,7 +58,7 @@ async function run() {
         namespace = 'default';
     }
     let action = core.getInput('action');
-    let manifests = manifestsInput.split('\n');
+    let manifests = manifestsInput ? manifestsInput.split('\n') : undefined;

```

Comment:
Is this required? `manifests` is a required input.

Reply:
Yeah. If you check a few lines above, manifestsInput is read but if its undefined, core.setFailed is called and execution continues, so it will fail here as undefined